app/containers/ITunesContainer/AllTracksContainer/index.js

```diff
@@ -0,0 +1,190 @@
+/**
+ *
+ * AllTracksContainer
+ *
+ */
+
+import React, { useState, useEffect, memo } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { injectIntl } from 'react-intl';
+import { debounce, isEmpty, get } from 'lodash';
+import styled from 'styled-components';
+import { injectSaga } from 'redux-injectors';
+import { createStructuredSelector } from 'reselect';
+import { compose } from 'redux';
+import { Link } from 'react-router-dom';
+import { Card, Input, Skeleton } from 'antd';
+import If from '@app/components/If';
+import T from '@app/components/T';
+import For from '@app/components/For';
+import ITunesCard from '@app/components/ITunesCard';
+import { selectITunesSearchQuery, selectITunesData, selectITunesError } from '../selectors';
+import { iTunesContainerCreators } from '../reducer';
+import saga from '../saga';
+
+const { Search } = Input;
+
+const CustomCard = styled(Card)`
+  && {
+    margin: 1.25rem 0;
+    max-width: ${(props) => props.maxwidth};
+  }
+`;
+const Container = styled.div`
+  && {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 1em;
+    padding: 1em;
+  }
+`;
+
+export function AllTracksContainer({
+  dispatchITunesSongs,
+  dispatchClearITunesSongs,
+  searchQuery,
+  iTunesError,
+  iTunesData,
+  intl,
+  maxwidth
+}) {
+  const [currentSongRef, setCurentSongRef] = useState();
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    const loaded = get(iTunesData, 'results', iTunesError);
+    if (loaded) {
+      setLoading(false);
+    }
+  }, [iTunesData, iTunesError]);
+
+  useEffect(() => {
+    if (searchQuery && !iTunesData?.results?.length) {
+      dispatchITunesSongs(searchQuery);
+      setLoading(true);
+    }
+  }, []);
+
+  const handleOnChange = (sQuery) => {
+    if (!isEmpty(sQuery)) {
+      dispatchITunesSongs(sQuery);
+      setLoading(true);
+    } else {
+      dispatchClearITunesSongs();
+    }
+  };
+
+  const handleOnActionClick = (trackRef) => {
+    if (!isEmpty(currentSongRef) && trackRef !== currentSongRef) {
+      currentSongRef?.current?.pause();
+    }
+    setCurentSongRef(trackRef);
+  };
+
+  const debouncedHandleOnChange = debounce(handleOnChange, 200);
+
+  const renderTrackDetails = () => {
+    const items = get(iTunesData, 'results', []);
+    const totalCount = get(iTunesData, 'resultCount', 0);
+    return (
+      <If condition={!isEmpty(items) || loading}>
+        <CustomCard>
+          <Skeleton loading={loading} active>
+            <If condition={!isEmpty(searchQuery)}>
+              <div>
+                <T id="search_term" values={{ searchQuery }} />
+              </div>
+            </If>
+            <If condition={totalCount !== 0}>
```

Comment:
!totalCount

Reply:
Made the change

Reply:
yes, my bad. Just `totalCount` will be alright since non zero numbers return truthy